src/components/Login.tsx

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import { useState } from 'react';
+
+import { Form, FormSection } from 'src/components/Form';
+import { Button, InlineFeedback, TextInput } from 'src/components/ui';
+import { useToast } from 'src/context/toast';
+
+export default function Login() {
+  const [email, setEmail] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+  const [isEmailValid, setIsEmailValid] = useState<boolean>(false);
+  const [isPasswordValid, setIsPasswordValid] = useState<boolean>(false);
+  const [shouldShowErrorState, setShouldShowErrorState] = useState(false);
+  const toast = useToast();
+
+  function validateEmail(value: string) {
```

Comment:
Let's convert all the `function`s inside a component to arrow functions `() => {}`.
```suggestion
  const validateEmail = (value: string) => {
    ...
   };
```

Basically only component functions should be defined as `export default function Login() {`. Rest of the functions should be arrow functions.

Reply:
Updated! Thank you.